src/utilities/mapUtilities.js

```diff
@@ -10,24 +10,28 @@ const generateConnectionData = (connection, tileId) => {
     return {
       osmStopId: connection[0].id.toString(),
       gtfsStopId: connection[1].id.toString(),
-      tileId: tileId
+      tileId: tileId,
     };
   } else if (connection[0].isOsm === false && connection[1].isOsm === true) {
     return {
       osmStopId: connection[1].id.toString(),
       gtfsStopId: connection[0].id.toString(),
-      tileId: tileId
+      tileId: tileId,
     };
   }
-  throw(new Error());
+  throw new Error();
 };
 
-const generateStopName = (id, name, number) => {
-  if (name) {
-    return `${name} ${number || ''}`;
-  } else {
-    return `${id}`;
+const generateStopName = stop => {
+  let result = stop.name || stop.id;
+  if (stop.number) {
+    result += ' ' + stop.number;
   }
+  if (stop.stopType === 1) return result;
+
+  const refTag = stop.tags.find(x => x.key === 'ref');
```

Comment:
imo it's more readable when you name item you iter by: (tagObject => tagObject.key === 'ref')

Reply:
In my opinion, it's less readable when this name is long. You should be more careful about proper list naming than list elements. You can call it an `item` but it'll always be a member of the list which is a simple value or an object. So if the list is called `tags` element will always be `tag`. 
The `Object` suffix is not necessary.
A long name can also cause overloading a code with that name. It happens when you use a name several times in an inline function.